include/Core/UI/Page/LogListPage.php

```diff
@@ -217,10 +217,13 @@ public function load_view_logs_assets( $hook ) {
 		$email_log      = email_log();
 		$plugin_dir_url = plugin_dir_url( $email_log->get_plugin_file() );
 
-		wp_enqueue_style( 'jquery-ui', $plugin_dir_url . 'assets/vendor/jquery-ui/themes/smoothness/jquery-ui.min.css', array(), '1.12.1' );
-		wp_enqueue_style( 'el-view-logs-css', $plugin_dir_url . 'assets/css/admin/view-logs.css', array( 'jquery-ui' ), $email_log->get_version() );
+		wp_enqueue_style( 'jquery-ui-css', $plugin_dir_url . 'assets/vendor/jquery-ui/themes/base/jquery-ui.min.css', array(), '1.12.1' );
+		wp_enqueue_style( 'el-view-logs-css', $plugin_dir_url . 'assets/css/admin/view-logs.css', array( 'jquery-ui-css' ), $email_log->get_version() );
 
 		wp_enqueue_script( 'jquery-ui-datepicker' );
-		wp_enqueue_script( 'el-view-logs-js', $plugin_dir_url . 'assets/js/admin/view-logs.js', array( 'jquery-ui-datepicker' ), $email_log->get_version(), true );
+		wp_enqueue_script( 'el-view-logs', $plugin_dir_url . 'assets/js/admin/view-logs.js', array( 'jquery-ui-datepicker' ), $email_log->get_version(), true );
+
+		wp_enqueue_script( 'jquery-ui', $plugin_dir_url . 'assets/vendor/jquery-ui/jquery-ui.min.js', array( 'jquery' ), $email_log->get_version() );
```

Comment:
Let's use the version number of jQuery UI and the other library instead of email log's version.